test/appium/tests/base_test_case.py

```diff
@@ -85,7 +85,7 @@ def capabilities_local(self):
         desired_caps['deviceName'] = 'nexus_5'
         desired_caps['platformName'] = 'Android'
         desired_caps['appiumVersion'] = '1.7.2'
-        desired_caps['platformVersion'] = '7.1'
+        desired_caps['platformVersion'] = '8.0'
```

Comment:
please remove the change, we do not expect to migrate on 8.0 version in scope of this PR